docs/payments-refunds/create-payments.md

```diff
@@ -36,18 +36,18 @@ curl https://api.mobilepay.dk/v1/payments \
 ```
 
 :::tip
-`mobilePayAppRedirectUri` should be used to redirect user to MobilePay app.
+`mobilePayAppRedirectUri` should be used to redirect a user to a MobilePay app.
 :::
 
 :::tip
-Use `description` to inform your customers to a greater extent. Description will be shown on the payment confirmation screen in MobilePay so the customer has more information about the product. The description will also be shown when the MobilePay user opens their payment receipt in the activity list.
+Use `description` to inform your customers to a greater extent. The description will be shown on the payment confirmation screen in MobilePay so the customer has more information about the product. The description will also be shown when the MobilePay user opens their payment receipt in the activity list.
```

Comment:
Is this a correct use of "the"? I imagine "Description" here is used as request property

Reply:
No, article is not needed here. I'll rollback.